index.d.ts

```diff
@@ -231,10 +231,79 @@ export interface LineSymbolizer extends BaseSymbolizer {
  */
 export type PointSymbolizer = IconSymbolizer | MarkSymbolizer | TextSymbolizer;
 
+/**
+ * A single entry for the ColorMap.
+ */
+export interface ColorMapEntry {
+  color: string;
+  quantity?: number;
+  label?: string;
+  opacity?: number;
+}
+
+/**
+ * The Types that are allowed in a ColorMap.
+ */
+export type ColorMapType = 'ramp'|'intervals'|'values';
```

Comment:
Minor: Missing spaces.

Reply:
solved in https://github.com/terrestris/geostyler-style/pull/113/commits/c6a661d54274315ef2f4c4c5f1fe23898b2f6d1f